Documentation/configuring-reporting-operator.md

```diff
@@ -70,6 +71,26 @@ And to authenticate using a username and password, use basic authentication:
 curl -u testuser:password123 -k "https://metering-openshift-metering.apps.example.com/api/v1/reports/get?name=cluster-memory-capacity-hourly&format=tab"
 ```
 
+#### Openshift Authentication
+
+When the following options are set to true, it enables authenticating using a bearer token from a serviceAccount or for your user when querying the reporting rest API:
```

Comment:
Slighty awkward sentence. Maybe try "When the following options are set to true, authentication using a bearer token is enabled for the reporting rest API. Bearer tokens may come from serviceAccounts or users."

Reply:
Yeah this sounds way better